crmsh/cibquery.py

```diff
@@ -0,0 +1,25 @@
+"""utilities for parsing CIB xml"""
```

Comment:
Maybe it's better to move these codes into xmlutil.py instead of creating a new py  file?

Reply:
No. xmlutil.py is too long, and its name does not reflect its content. Instead of general xml processing utilities, it is a mixed mess with shell utils, CIB specific routines, and even CLI handlers.